### PR TITLE
Simplify Template

### PR DESCRIPTION
**Pull request by archiewood**
Run through to improve the template for first user landing experience

- hero chart above the fold
- headers to [outcome] with [feature]
- less content
- 10x smaller duckdb
- no emojis on front page
- CTA to get online w Cloud

| Before  | After |
| --- | --- |
| CleanShot 2023-08-23 at 13 02 23@2x | CleanShot 2023-08-23 at 13 00 29@2x |

### CODE REVIEW

**Comment by archiewood** [comment]
Also see #82

**Comment by hughess** [comment]
This is looking great!

- I like the switch to `fmt` props - that's a nice intro for a new user
- Are there other props we can include to signal that things are changeable? Fmt is definitely the best, so we might not need others, but overriding a column title in the data table or changing a bar chart colour are other ideas
- The appearance looks clean, but maybe a bit plain - would it make sense to include some more visually interesting features to the components like annotations and/or comparisons in BigValues?
- Are there any UI components we want to show off? Maybe tabs? Or maybe UI components should be a section similar to data viz components
- It would be great to include a templated page - or at least a reference to the concept (I think under the control output section and the separate page) 
   - E.g., "or programmatically generate pages from your data using templated pages", "or use templated pages to generate a series of pages from a single markdown template - e.g., 1 page per customer"
- I think a Details component would fit quite nicely in here, maybe under the hero chart, with a description that you can include documentation directly in your report/alongside the viz 
- The sidebar looks a bit sparse with only the one link there. Are there other pages we could/should add - one idea is a help page with the community links
- Do the queries show as hidden by default?
   - When the page references the "query above", I don't think people will know what it's referring to if the query is hidden + I don't think they'll naturally find the show queries button from there 
- A few wording ideas
   - Show Data with Components --> "Visualize Data with Components"?
   - "Share Data" - to me, sharing data means sharing the actual datasets. Would something like "project", "reports", or "app" work there?
   - "Control output" - this might be a good place to include words like "programmatically" or "dynamic", but not sure exactly where - just a thought

**Comment by archiewood** [comment]
Great feedback. 
- [x] More customizable props
- [x] Reference area for second barchart. (Held back on BigValue comparison, as I think the SQL you end up adding adds to the code complexity)
- [x] Wording edits


> Do the queries show as hidden by default?

No, in dev they are shown by default. In prod they are hidden by default.


## Other suggestions
- Param pages
- num of items in sidebar
- UI components
- Details component

I'm trading off brevity vs expansiveness here.

Things I like about current version
- Less that 100 lines of code
- Only two files to delete if you know what you are doing and just want to get going
- Still looks mainly like markdown. It's not got so many custom component tags that it looks like something else

**Comment by archiewood** [comment]
Decision: Going to leave templated pages out!

Feel like highly likely user would have deleted this page by the time they got around to needing the example for the feature

File: pages/index.md [hunk 1]
```diff
@@ -1,101 +1,88 @@
...
 select
   date_trunc('month', order_datetime) as order_month,
   count(*) as number_of_orders,
-  sum(sales) as sales_usd0k,
-  sum(sales)/count(*) as average_order_value_usd2
+  sum(sales) as sales,
+  sum(sales)/count(*) as average_order_value
 from orders
-
+where order_datetime >= '2020-01-01'
```
Comment (archiewood) [inline pages/index.md:28]: I think this is easier to understand

Comment (hughess) [reply pages/index.md:28]: Shouldn't this be `>= 2021` for this report?

Comment (archiewood) [reply pages/index.md:28]: good spot

**Pull request merged**